lib/pager_api.rb

```diff
@@ -4,6 +4,14 @@ module PagerApi
   mattr_accessor :pagination_handler
   @@pagination_handler = :kaminari
 
+  # page access method
+  mattr_accessor :page_access_method
+  @@page_access_method = :page
+
+  # per page access method
+  mattr_accessor :per_page_access_method
+  @@per_page_access_method = :per
```

Comment:
Maybe we can rename this to `per_page` to match a default value for Kaminari and WillPaginate, how does that sound?